crawler/crawler/bitbucket.go

```diff
@@ -251,17 +246,18 @@ func RegisterBitbucketAPI() OrganizationHandler {
 
 		// if last page for this organization, the result.Next is empty.
 		if len(result.Next) == 0 {
-			return "", nil
+            return nil, nil
 		}
 
 		// Return next url.
-		return result.Next, nil
+		u, _ := url.Parse(result.Next)
```

Comment:
[**warning**] The avoid of logging or returning error in URL Parsing is wanted?

Reply:
This one will be removed as soon as we use bitbucket's go library.

Reply:
perfect!